src/pages/RealmScans/main.ts

```diff
@@ -0,0 +1,76 @@
+import { pageInterface } from '../pageInterface';
+
+export const RealmScans: pageInterface = {
+  name: 'RealmScans',
+  domain: 'https://realmscans.com',
+  languages: ['English'],
+  type: 'manga',
+  isSyncPage(url) {
+    return j.$('#screenmode').length > 0;
+  },
+  isOverviewPage(url) {
+    return utils.urlPart(url, 3) === 'series';
+  },
+  getImage() {
+    return j.$('div.thumb > img').attr('src');
+  },
+  sync: {
+    getTitle(url) {
+      return j.$('ol > li:nth-child(2) > a > span').text();
+    },
+    getIdentifier(url) {
+      const temp_ident = j.$('ol > li:nth-child(2) > a').attr('href') || '';
```

Comment:
Could be replaced with 
`return RealmScans.overview!.getIdentifier(RealmScans.sync.getOverviewUrl(url));`